### DISCUSSION TITLE
Can anybody share their paramter for ContentDetector that work well and detecting scenes of raw unedited footage?

### DISCUSSION THREAD

**Discussion by apthomas** [Breakthrough/PySceneDetect#371] [Detection Algorithms]
I've begun playing around with PySceneDetect and only just the threshold attribute, but would be curious if others have tuned the ContentDetector parameters so that performance is good on unedited raw footage?

Additionally if you have, do you know of datasets with raw footage and labeled scenes that allow you to use as ground truth in evalauting the performance of the ContentDetector with said attributes?

**Comment by Breakthrough** [comment]
I've found research papers that reference PySceneDetect to be useful for finding different datasets to use for testing purposes.  For example:
https://github.com/Tangshitao/ClipShots

This is definitely an open area of research for the project, especially now that the latest version of PySceneDetect allows you to customize the weights applied to the different components used for calculating the content score.  It would be awesome if evaluation of different detectors and parameters could be automated using one of these datasets, especially with [other detection methods in the pipeline](#295).

**Reply by Breakthrough** [reply]
It was also brought up in #226 that the threshold could be detected automatically.  I think this could be done with some kind of online outlier detection algorithm.  My initial feeling was a statistical approach was the best, but that might not be required for this anyways so long as we have decent heuristics on what metrics to use and when.
